src/Mappers/InventoryMapper.php

```diff
@@ -110,19 +111,50 @@ public function createInventoryDTOsFromVariation($variationData)
         continue;
       }
 
+      // Avl Immediately. Net Stock.
+      $onHand = isset($stock['netStock']) ? $stock['netStock'] : null;
+      // Already Ordered items.
+      $onOrder = isset($stock['reservedStock']) ? $stock['reservedStock'] : null;
+
+      // key for a potential preexisting DTO that we need to merge with
+      $dtoKey = $supplierId . '_' . $supplierPartNumber;
+
+      /** @var RequestDTO $existingDTO */
+      $existingDTO = $inventoryDTOs[$dtoKey];
+
+      if (isset($existingDTO))
+      {
+        // merge with previous values for this suID
+        if (isset($existingDTO->getQuantityOnHand()))
+        {
+          $onHand += $existingDTO->getQuantityOnHand();
+        }
+
+        if (isset($existingDTO->getQuantityOnOrder()))
+        {
+          $onOrder += $existingDTO;
```

Comment:
This should be $onOrder += $existingDTO->getQuantityOnOrder();

Reply:
oh. good call.